modules/core/src/test/java/org/apache/ignite/internal/testframework/WorkDirectoryExtensionTest.java

```diff
@@ -380,7 +380,7 @@ void test() {
     /**
      * Tests {@code WorkDirectoryExtension} lifecycle works correctly on a test class with all test methods being disabled.
      *
-     * @see <a href="https://issues.apache.org/jira/browse/IGNITE-15799">IGNITE-15799</a>
+     * @see <a href="https://issues.apache.org/jira/browse/IGNITE_15799">IGNITE_15799</a>
```

Comment:
Invalid link.

Reply:
this change is intentional - there is a desire to avoid linking in a code to resolved tickets because we have a bot that checks it. It looks a crutch and potentially in the future will be changed.

Reply:
I think we should either to fix the bot or remove links to resolved tickets.
Why do we need a link to a ticket if we can just write comprehensive comment instead?

Reply:
removed from the patch